platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/consensus/ConsensusRounds.java

```diff
@@ -37,7 +37,7 @@ public class ConsensusRounds {
     /** consensus configuration */
     private final ConsensusConfig config;
     /** stores the minimum generation for all decided and non-expired rounds */
-    private final SequentialRingBuffer<MinGenInfo> minGenStorage;
+    private final SequentialRingBuffer<MinimumJudgeInfo> minGenStorage;
```

Comment:
Optional: this member variable name, and many comments/parameter names in this class could also be updated.

Reply:
I renamed the variables and methods, and I fixed the javadocs. I specifically did not alter the business logic that does stuff like compare `event.getGeneration()`, as that is outside the scope of this PR.